fb_apt/attributes/default.rb

```diff
@@ -0,0 +1,22 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+
+if node.debian?
+  mirror = 'http://http.debian.net/debian'
```

Comment:
nope, and the 404 is expected. http://http.debian.net is a magic redirecting mirror that sends you to the closest one to you.